.github/workflows/build_release.yml

```diff
@@ -141,23 +141,3 @@ jobs:
           file: scaler/Dockerfile
           context: .
           push: true
-
-  e2e_tests:
```

Comment:
Why are we removing it here?

Reply:
Because now they are executed with the new check (every PR and every push to main), using the manifests inside the repo instead of helm. I can add them here again using the manifests in the repo if you think is worth, but I don't like to be attached to helm chart